app/app/signup/page.jsx

```diff
@@ -391,6 +391,242 @@ const SignUp = () => {
+                  borderColor: theme === "dark" ? "#444444" : "#5A4A42",
+                },
+              }}
+            />{" "}
+            <TextField
+              variant="outlined"
+              margin="normal"
+              required
+              fullWidth
+              name="password"
+              label="Password"
+              type="password"
+              id="password"
+              autoComplete="new-password"
+              value={data.password}
+              onChange={handleChange}
+              error={!!errors.password}
+              helperText={errors.password}
+              onBlur={handleBlur}
+              sx={{
+                "& .MuiOutlinedInput-root": {
+                  borderRadius: "10px",
+                  bgcolor: theme === "dark" ? "#333333" : "#fffbe8",
+                  color: theme === "dark" ? "#ffffff" : "#3E2723",
+                },
+                "& .MuiInputLabel-root": {
+                  color: theme === "dark" ? "#bbbbbb" : "#5A4A42",
+                  fontFamily: "serif",
+                },
+                "& .MuiOutlinedInput-notchedOutline": {
+                  borderColor: theme === "dark" ? "#444444" : "#5A4A42",
+                },
+              }}
+            />{" "}
+            <Button
+              type="submit"
+              fullWidth
+              variant="contained"
+              color="primary"
+              sx={{
+                background: theme === "dark" ? "#666666" : "grey",
+                mt: 3,
+                fontFamily: "jacques Francois",
+                textTransform: "none",
+                fontSize: "20px",
+              }}
+              onClick={handleSubmit}
+            >
+              Submit
+            </Button>{" "}
+            <Typography
+              variant="h8"
+              sx={{
+                mt: 2,
+                color: theme === "dark" ? "#ffffff" : "#3C3C3C",
+                textAlign: "center", // Center-align text
+                fontFamily: "jacques Francois",
+              }}
+            >
+              Already have an account?{" "}
+              <a href="/login" target="_blank" style={{ color: "blueviolet" }}>
+                Login
+              </a>
+            </Typography>{" "}
+            <Button
+              variant="text"
+              sx={{
+                mt: 8, // Add spacing between the buttons
+                color: theme === "dark" ? "#bbbbbb" : "#5A4A42",
+                fontFamily: "jacques Francois",
+                fontSize: "16px",
+              }}
+              onClick={() => router.push("/")}
+            >
+              Back to Home
+            </Button>
+          </Container>{" "}
+        </Container>
+      </Container>
+      <ErrorModal
+        isOpen={error.isOpen}
+        onClose={hideError}
+        message={error.message}
+        title={error.title}
+        severity={error.severity}
+      />
+    </AppLayoutContainer>
+  );
+=======
```

Comment:
and here

Reply:
Yes, I messed with merge conflicts. I realized and fixed everything. Thank you for your time